website/content/en/docs/concepts/node-templates.md

```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
 </powershell>
 ```
 
+{{% alert title="Note" color="primary" %}}
+Currently, Karpenter does not specify `-ServiceCIDR` to [EKS Windows AMI Bootstrap script](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html#bootstrap-script-configuration-parameters).
+Windows worker nodes will use `172.20.0.0/16` or `10.100.0.0/16` for Kubernetes service IP address ranges based on the IP address of the primary interface.
+
+The effective ServiceCIDR can be verified at `$env:ProgramData\Amazon\EKS\cni\config\vpc-bridge.conf` on the worker node.
+If the effective ServiceCIDR is incorrect for your windows worker nodes, you can add below userData as workaround.
```

Comment:
Looks good. Please add the link to the Github issue for ServiceCIDR and add a statement that the feature is coming soon, but this is the workaround for now.

Reply:
Updated to mention the ongoing feature issue.